app/main.py

```diff
@@ -1,8 +1,25 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
 
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        self.__class__.people[name] = self
 
-def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+
+def create_person_list(people_list: list) -> list:
+    person_list = []
+    for prs in people_list:
```

Comment:
prs is a bad name for variable, use full names for naming

Reply:
ok, I changed the name. 
But in this case, the short name was intentional because there was
class = Person,
class attribute = people,
list = people
and with such a name, the variable in the for loop confuses all of this code for me finally.